### PR TITLE
Checks if SIWE message is expired

### PR DESCRIPTION
**Pull request by vzotova**

### CODE REVIEW

File: src/fragments/chatcontent.tsx [hunk 1]
```diff
@@ -4,10 +4,11 @@ import { SingleSignOnEIP4361AuthProvider, USER_ADDRESS_PARAM_EXTERNAL_EIP4361 }
 import { decodeB64 } from "../../utils/common";
 import { useCeramicContext } from "../../context";
 import { decryptWithTACo, parseUrsulaError } from "../../utils/taco";
-import { authenticateCeramic, alreadyLoggedIn, getCeramicSiweInfo } from "../../utils";
+import { authenticateCeramic, alreadyLoggedIn, getCeramicSiweInfo, reset } from "../../utils";
```
Comment (derekpierre) [inline src/fragments/chatcontent.tsx]: Does `reset` still need to be imported?

File: src/fragments/chatcontent.tsx [hunk 2]
```diff
@@ -4,10 +4,11 @@ import { SingleSignOnEIP4361AuthProvider, USER_ADDRESS_PARAM_EXTERNAL_EIP4361 }
...
 import { Message } from "../../types";
 import Avatar from "./avatar";
 import Spinner from "~/fragments/spinner";
+import { SiweMessage } from 'siwe';
```
Comment (derekpierre) [inline src/fragments/chatcontent.tsx]: Is this import still necessary?

File: src/fragments/chatcontent.tsx (hunk 1 above)
Comment (vzotova) [reply src/fragments/chatcontent.tsx]: thanks!
I've got used to pre commit hooks so forgot to check by myself what to clean.
✔️

File: src/fragments/chatcontent.tsx (hunk 2 above)
Comment (vzotova) [reply src/fragments/chatcontent.tsx]: ✔️

**Review by derekpierre — APPROVED** [review]
🎸

**Pull request merged**